eng/versioning/version_client.txt

```diff
@@ -103,38 +103,38 @@ com.azure:azure-storage-internal-avro;12.0.1;12.0.2-beta.2
 com.azure:azure-storage-perf;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-storage-queue;12.7.0;12.8.0-beta.2
 com.azure:perf-test-core;1.0.0-beta.1;1.0.0-beta.1
-com.azure.spring:azure-identity-spring;1.0.0-beta.1;1.0.0-beta.1
-com.azure.spring:azure-spring-boot-starter-active-directory-b2c;3.0.0-beta.1;3.0.0-beta.2
-com.azure.spring:azure-spring-boot-starter-active-directory;3.0.0-beta.1;3.0.0-beta.2
-com.azure.spring:azure-spring-boot-starter-cosmos;3.0.0-beta.1;3.0.0-beta.2
+com.azure.spring:azure-identity-spring;1.0.0;1.0.0
+com.azure.spring:azure-spring-boot-starter-active-directory-b2c;3.0.0-beta.2;3.0.0
```

Comment:
There is no `3.0.0-beta.2` in repo.

https://repo.maven.apache.org/maven2/com/azure/spring/azure-spring-boot-starter-active-directory-b2c/

Reply:
We will recover to 3.0.0-beta.1